lib/stripe/api_operations/request.rb

```diff
@@ -122,6 +127,11 @@ def self.included(base)
         )
       end
 
+      private def request_stripe_object(method:, path:, params:, opts: {})
```

Comment:
For ListObject, it's not a resource but needs to make requests.

Reply:
Got it - hmmm this does get a bit confusing since APIResource pulls in `Request` (https://github.com/stripe/stripe-ruby/blob/001db8bd/lib/stripe/api_resource.rb#L5) and also has its own instance and class versions of this method...

Reply:
Yeah, API resource overrides the instance implementation, it is a bit convoluted.